bad-ranges.js

```diff
@@ -41,6 +41,8 @@ const EXPERIMENTAL_BAD_RANGES = [
   // This was a regression from https://github.com/web-platform-tests/wpt/pull/29089,
   // fixed by https://github.com/web-platform-tests/wpt/pull/32540
   [moment('2022-01-25'), moment('2022-01-27')],
+  // This was a very much incomplete Safari run.
+  [moment('2023-09-01'), moment('2023-09-03')],
```

Comment:
```suggestion
  [moment('2023-09-02'), moment('2023-09-03')],
```

no?

Reply:
I was just staying on the safe side, but this works.